src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/order/history.yaml

```
@@ -51,21 +51,73 @@ sylius_twig_hooks:
             addresses:
                 template: '@SyliusAdmin/order/history/content/sections/addresses.html.twig'
                 priority: 0
-
+                                     
         'sylius_admin.order.history.content.sections.addresses':
             billing_address:
                 component: sylius_admin:order:address_history
                 props:
                     addressId: '@=_context.resource.getBillingAddress().getId()'
                     header: 'sylius.ui.billing_address'
                     sort: 'desc'
-                    template: '@SyliusAdmin/order/history/content/sections/addresses/address.html.twig'
+                    template: '@SyliusAdmin/order/history/content/sections/addresses/billing_address.html.twig'
                 priority: 100
             shipping_address:
                 component: sylius_admin:order:address_history
                 props:
                     addressId: '@=_context.resource.getShippingAddress().getId()'
                     header: 'sylius.ui.shipping_address'
                     sort: 'desc'
-                    template: '@SyliusAdmin/order/history/content/sections/addresses/address.html.twig'
-                    priority: 0
+                    template: '@SyliusAdmin/order/history/content/sections/addresses/shipping_address.html.twig'
+                priority: 0
+
+        'sylius_admin.order.history.content.sections.addresses.billing_address':
+            header: 
+                template: '@SyliusAdmin/order/history/common/sections/addresses/header.html.twig'
```

```suggestion
                template: '@SyliusAdmin/order/history/common/sections/addresses/address/header.html.twig'
```
Apply this to all paths below